packages/@okta/vuepress-theme-prose/assets/css/components/_onThisPage.scss

```diff
@@ -10,6 +10,7 @@
 
     @include media('>=tablet') {
       top: 150px;
+      display: none;
```

Comment:
Why are we hiding this section in desktop views?

Reply:
We definitely should not be. On This Page should continue to be available in the right sidebar on desktop.

Reply:
It is hidden in Figma for larger than tablet screens

Reply:
Which Figma mockup were you looking at? It doesn't look like we have a mockup for wide desktops.

Reply:
Mockup for 1024 px